chainer/datasets/tox21.py

```diff
@@ -0,0 +1,150 @@
+import os
+import shutil
+import warnings
+import zipfile
+
+
+from chainer.dataset import download
+from chainer.datasets import tuple_dataset
+import numpy
```

Comment:
Please put `import numpy` in a new chunk between standard imports and chainer imports.